Tools/Scripts/libraries/webkitscmpy/webkitscmpy/local/git.py

```diff
@@ -609,6 +609,14 @@ def branches_for(self, hash=None, remote=True):
             return sorted(result.get(remote, []))
         return result
 
+    def is_pull_request_branch(self, branch, source_remote):
```

Comment:
Bikeshed: `is_pull_request_branch` seems like a slightly bad name, it makes me think that this tells you whether the branch is _already_ a pull-request. Maybe `is_suitable_branch_for_pull_request`?

Also a nit, which I don't expect us to address here, but a discussion we should have at some point: `source_remote`, while I realise what we use throughout `webkitscmpy`, is also slightly confusing — is that what GitHub calls the "base" or the "head" remote? I'm pretty sure the answer is the "base" repository (i.e., where the PR will be merged into), rather than the "head" repository (i.e., where the PR will be merged from), though to me "source" seems more like "where you're going from".

Reply:
Personally I link `source` with "source of truth" rather than the "going from" sense but could be argued either way..